src/clients/api/queries/useGetMainAssets/index.ts

```diff
@@ -183,22 +194,23 @@ const useGetMainAssets = ({
           supplierCount: market?.supplierCount || 0,
           borrowerCount: market?.borrowerCount || 0,
           supplyBalanceTokens: new BigNumber(market?.totalSupply2 || 0).div(exchangeRateVTokens),
+          supplyBalanceCents: convertDollarsToCents(
+            market?.totalSupplyUsd ? +market?.totalSupplyUsd : 0,
+          ),
           borrowBalanceTokens: new BigNumber(market?.totalBorrows2 || 0),
-          supplyBalanceCents: new BigNumber(market?.totalSupplyUsd || 0)
-            .times(100)
-            .dp(0)
-            .toNumber(),
-          borrowBalanceCents: new BigNumber(market?.totalBorrowsUsd || 0)
-            .times(100)
-            .dp(0)
-            .toNumber(),
+          borrowBalanceCents: convertDollarsToCents(
+            market?.totalBorrowsUsd ? +market?.totalBorrowsUsd : 0,
```

Comment:
It is yes, because `totalBorrowsUsd` is a string so we convert it to a number. However you highlighted that the second optional chaining is unnecessary since we've already assessed that `market.totalBorrowsUsd` isn't `undefined` hehe

I'll update that.